packages/theme-chalk/src/table.scss

```diff
@@ -34,6 +34,12 @@
     }
   }
 
+  tbody {
+    &:focus {
+      outline: none;
+    }
+  }
+
```

Comment:
The style here seems unnecessary to add.

Reply:
Sorry, this should not be removed directly, but override the `outline` style of the `focus-visible` pseudo-class.

![image](https://github.com/element-plus/element-plus/assets/24516654/87ac47a6-1935-44ae-8fad-0f632e45b0f2)

Reply:
agree @FrankFang

Reply:
Any suggestions? I am not sure whether `outline: none;` affects the other components' styles.
But removing `outline: none` makes the table component weird.
For now, I just add `outline: none` back.

Reply:
You should only need to add the following styles: 
```scss
tbody {
  &:focus-visible {
    outline: none;
  }
}
```

Reply:
It makes sense. I've pushed my code.